lib/smart_answer/calculators/marriage_abroad_calculator.rb

```diff
@@ -334,6 +334,10 @@ def path_to_outcome
                    'opposite_sex'
                  end
 
+      if ceremony_country == 'philippines'
```

Comment:
I would like to refactor this method to something like:

```ruby
if calculator.is_two_outcomes_country?
  [ceremony_country_path, same_sex_path]
else
  [ceremony_country_path, ceremony_location_path, partner_nationality_path, same_sex_path]
```

As there are a few countries already in fact-check [I have created a Trello card][1] to perform the refactor.

[1]: https://trello.com/c/V1T9dLEg/481-refactor-for-marriage-abroad-two-outcomes